src/config/konnectors.json

```diff
@@ -808,6 +808,37 @@
       "bankId": "109"
     }
   },
+  {
+    "slug": "carrefour159",
```

Comment:
It is maybe to late but why using this slug instead of `carrefourbanque` ?

Reply:
Exactly